_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,217 @@
+    half-signed with `SIGHASH_SINGLE|SIGHASH_ANYONECANPAY`.  Fournier
+    extended the previous half-signed proposal and then provided his own
+    proposal that was equivalently effective but simpler.  The new
+    proposal has the initiator create and sign (but not broadcast) a
+    transaction that spends their UTXO back to themselves.  They give
+    this to the other party as a proof of good faith---if the initiator
+    later fails to sign the actual funding transaction, the respondent
+    can broadcast the good-faith transaction, forcing the initiator to
+    pay an onchain fee.  Fournier concludes his post with a summary of
+    the tradeoffs between the different approaches.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Add unit testing of node eviction logic][review club
+#20477] is a PR ([#20477][Bitcoin Core #20477]) by
+pseudonymous contributor practicalswift to improve test coverage of Bitcoin Core's peer
+eviction logic when a node's inbound connection slots are full. Extreme care
+must be taken to avoid exposing the node to attacker-triggered network
+partitioning with this logic, so this code is critical.
+
+Most of the discussion focused on understanding Bitcoin Core's peer eviction
+logic.
+
+{% include functions/details-list.md
+  q0="Inbound or outbound peer selection: which one is our primary defense
+      against eclipse attacks?"
+  a0="Outbound peer selection, as outbound peers are far less under attacker
+      control than inbound connections. Inbound peer eviction is a second-order
+      protection---and not all nodes permit incoming connections."
+  a0link="https://bitcoincore.reviews/20477#l-77"
+
+  q1="Why does Bitcoin Core evict inbound connections?"
+  a1="To make inbound slots available for honest peers in the network so that
+      new nodes can establish good outbound connections to them. Otherwise,
+      dishonest nodes could more easily attack new nodes by being available for
+      their outbound connections and by statically occupying inbound slots."
+  a1link="https://bitcoincore.reviews/20477#l-66"
+
+  q2="When it needs to free up a slot, how does Bitcoin Core decide which
+      inbound peer to evict?"
+  a2="Up to 28 peers are protected from eviction based on criteria that are
+      difficult to forge, such as low latency, network group, providing novel
+      valid transactions and blocks, and a few others. The longest-connected
+      remaining half are protected, including some onion peers. Of those that
+      remain, the youngest member of the network group with the most
+      connections is selected for disconnection. An attacker would have to be
+      simultaneously better than honest peers at all of these characteristics
+      to partition a node."
```

Comment:
This seems like a little bit too much detail. I'd suggest perhaps giving a higher-level answer here.

Reply:
This highlights a limitation of the review club format. I felt I was overly simplifying and yet don't touch the changes made by the PR other than "adds test coverage," because the details of the PR itself aren't particularly newsworthy or quiz-worthy.

Reply:
I'm not worried about excess detail when the answer content is hidden by default and readers need to click to expand.